### ISSUE TITLE
Trailer menu is covered by menubar when opened if "automatically hide menubar" option is selected

### ISSUE THREAD

**Issue by ptsochantaris** [ptsochantaris/trailer#228]

**Comment by ptsochantaris** [comment]
After following this up, it seems that NSScreen is correctly being accessed and the metrics of the screen are correctly being read. It's just that, since the menu bar is considered transient, the visible frame IS actually the whole height of the screen.

Having used this a bit more, it seems that NSScreen is reporting the right thing - as soon as the mouse is moved off the menubar, Trailer would look weird with an empty space above it. It only feels weird if you don't actually use menubar hiding :)

In short, I over-thinked this - nothing to see here :)
